contrib/macdeploy/README.md

```diff
@@ -11,5 +11,5 @@ This script should not be run manually, instead, after building as usual:
 During the process, the disk image window will pop up briefly where the fancy
 settings are applied. This is normal, please do not interfere.
 
-When finished, it will produce `Bitcoin-Core.dmg`.
+When finished, it will produce `Bitcoin-Qt.dmg`, `Bitcoin Core.app` and `dist/Bitcoin Core.app`.
```

Comment:
Two `.app`s? Is this intentional?

Reply:
I think it'd make sense to make this a list, and describe what each item is

Reply:
The two `.app` files were intentional as both are created, one in the root folder and the other is in the `dist`.

I am unsure of the differences between these two `.app` files, so any insight into this would be appreciated

Reply:
OK—I think there's a misunderstanding about the purpose of this documentation. The idea here is not to list every intermediate that will be included in the image, but only the end product of the macdeploy (the `.dmg` image). If the purpose of the intermediate files is not clear, better to not list them.

Reply:
Okay thanks @laanwj, I will reverse the change back to just listing the `.dmg` file as suggested and keep the filename change to match what is actually produced from `make deploy` (`Bitcoin-Qt.dmg`)

Reply:
Done and squashed the commits 👍